### PR TITLE
Fix errors for UG Draft

### PR DESCRIPTION
**Pull request by Eugenetanwl3881**
Fixed some errors in details and formatting which I found in the UG to tidy up for UG draft submission

### CODE REVIEW

File: docs/_ug/ItemsTags.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ The following are the attributes stored for each item:
 * Item remark
 * Item tags
 
-FoodRem items are unique by name and case-insensitive. 
+FoodRem items are unique by name and case-sensitive.
```
Comment (RichDom2185) [inline docs/_ug/ItemsTags.md:17]: Are they case sensitive? I thought `potato` and `Potato` would be considered the same item...

File: docs/_ug/ItemsTags.md [hunk 2]
```diff
@@ -14,15 +14,15 @@ The following are the attributes stored for each item:
...
+FoodRem items are unique by name and case-sensitive. 
 
 This means you cannot add two items on the same name.
 
 Restrictions for all attributes can be found in the [Placeholders](#placeholders) section.
 
 #### Tag
 
-A Tag in FoodRem serves as a means to categorise items. These tags are also unique and case-insensitive.
+A Tag in FoodRem serves as a means to categorise items. These tags are also unique and case-sensitive.
```
Comment (RichDom2185) [inline docs/_ug/ItemsTags.md:25]: Ditto on case sensitivity.

File: docs/_ug/commands/GeneralCommands.md [hunk 3]
```diff
@@ -18,8 +18,8 @@ help help
 
 A new window will open showing instructions.
 
-If no `COMMAND_WORD` was specified, only a general help will be provided.
-The general help shows a list of commands available to the user and a URL to this User Guide.
+If no `COMMAND_WORD` was specified, only a general help message will be provided.
+The general help message shows a list of commands available to the user and a URL to this User Guide.
```
Comment (RichDom2185) [inline docs/_ug/commands/GeneralCommands.md:22]: Let's remove hard breaks.

```suggestion
If no `COMMAND_WORD` was specified, only a general help message will be provided. The general help message shows a list of commands available to the user and a URL to this User Guide.
```

File: docs/_ug/commands/TagCommands.md [hunk 4]
```diff
@@ -114,8 +111,9 @@ Tags: {}
 
 > Renames a tag currently in FoodRem.
 
-**Note:**
-* The first `TAG_NAME` in the command refers to the current tag you wish to rename while the second `TAG_NAME` refers to the new name you wish to rename the current tag to.
+```note
```
Comment (RichDom2185) [inline docs/_ug/commands/TagCommands.md:114]: An `info` would probably be better here.

```suggestion
```info
```

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks!

File: docs/_ug/ItemsTags.md (hunk 1 above)
Comment (Eugenetanwl3881) [reply docs/_ug/ItemsTags.md:17]: Let me double-check with the rest. Based on our current implementation on master, `Potato` and `potato` are treated as different items.

Comment (Eugenetanwl3881) [reply docs/_ug/ItemsTags.md:17]: Ok, current implementation is case-sensitive. So, on the UG, we'll write it as case-sensitive first. Should we (and when we) update the implementation we can change this

Comment (RichDom2185) [reply docs/_ug/ItemsTags.md:17]: Noted, thanks!

**Pull request merged**